### PR TITLE
Refactor and cleanup PKGBUILDS

### PR DESCRIPTION
**Pull request by sirlucjan**
"Small" refactoring of PKGBUILD.

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>

### CODE REVIEW

File: linux-bore/PKGBUILD [hunk 1]
```diff
@@ -311,6 +311,9 @@ prepare() {
     echo "Setting config..."
     cp ../config .config
 
+    echo "Setting config..."
```
Comment (ptr1337) [inline linux-bore/PKGBUILD:314]: We dont need it doubled.
Please remove the doubled "Settings config..."

File: linux-cachyos-bmq/PKGBUILD [hunk 2]
```diff
@@ -311,6 +311,9 @@ prepare() {
     echo "Setting config..."
     cp ../config .config
 
+    echo "Setting config..."
```
Comment (ptr1337) [inline linux-cachyos-bmq/PKGBUILD:314]: Same

**Review by ptr1337 — COMMENTED** [review]
Please remove the doubled setting config.

Besides that fine

File: linux-cachyos-bmq/PKGBUILD (hunk 2 above)
Comment (sirlucjan) [reply linux-cachyos-bmq/PKGBUILD:314]: Clearly, added by mistake. There will be a correction in a moment

File: linux-bore/PKGBUILD (hunk 1 above)
Comment (sirlucjan) [reply linux-bore/PKGBUILD:314]: Sure, I will fix.

File: linux-cachyos-bmq/PKGBUILD (hunk 2 above)
Comment (sirlucjan) [reply linux-cachyos-bmq/PKGBUILD]: Done

File: linux-bore/PKGBUILD (hunk 1 above)
Comment (sirlucjan) [reply linux-bore/PKGBUILD]: Done

**Review by ptr1337 — APPROVED** [review]
Everything fine!

LGTM

**Review by ventureoo — APPROVED** [review]
More cases to the god of cases. LGTM.

**Pull request merged**